prow/plugins.yaml

```diff
@@ -5,12 +5,17 @@
 triggers:
 - repos:
   - istio/api
-  - sebastienvas/core
+  - istio/common-files
```

Comment:
Best practice here is to trigger at the `istio` and `istio-releases` org level, rather than each repo individually.

Historically that would have caused issues if the repos didn't have any prow jobs (I think comment spam). It is no longer the case. 

This is how we configure the kubernetes org:
* https://github.com/kubernetes/test-infra/blob/d4a5312e1597a3be2d8f0ae96ca4c40e94b092a0/prow/plugins.yaml#L5-L12
* https://groups.google.com/d/msg/kubernetes-dev/D8_6DtuhyS4/IQHYNXqRBwAJ

Reply:
Thanks, updated.